examples/data-objects/table-tree/tests/table.test.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { globals } from "../jest.config.cjs";
+
+describe("table", () => {
+	beforeAll(async () => {
+		await page.goto(globals.PATH, { waitUntil: "load", timeout: 0 });
+		await page.waitForFunction(() => window["fluidStarted"]);
+	}, 45000);
+
+	beforeEach(async () => {
+		await page.goto(globals.PATH, { waitUntil: "load" });
+		await page.waitForFunction(() => window["fluidStarted"]);
+	});
+
+	it("The page loads and there's a textbox with placeholder Row ID", async () => {
+		await expect(page).toClick("button", { text: "Add Row" });
+	});
```

Comment:
Nit: I would probably name this more a bit more generally so that we don't have to update the test name if/when implementation details change. E.g.
```suggestion
	it("The page loads", async () => {
		// Verify that our "add row" UI exists on the page as a simple smoke test to verify that the page has been loaded correctly
		await expect(page).toClick("button", { text: "Add Row" });
	});
```